docs/developing/integrate/wallets/safe.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Safe
+proofedDate: 20240609
+iterationBy: na
+includedInSite: false
+approvedBy: na
+comment:
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+
+What is Safe?
+
+The Safe Account functions as a proxy contract, which means it forwards requests to a singleton contract. This singleton contract holds all the essential logic required for the Safe to operate. Proxy contracts are advantageous because they are typically more cost-effective to deploy and manage on public blockchains and they allow for easier upgrades. The primary purpose of the Safe proxy contract is to oversee and maintain the Safe’s state.
+The term "contract state" refers to the variables (or values) associated with a specific contract. Functions and contract code are usually stored and executed as bytecode within the Neon EVM, while variables are managed separately. These variables are the values that functions change when they run. For the Safe, the singleton contract provides the functions that adjust the state variables in the proxy. This singleton contract is also called the master copy of the Safe.
+
+
+## Prerequisites
+
+- NodeJS 16.x or newer
+
+## How to integrate Web3Auth SDK
```

Comment:
The heading here should be changed like - "How to create a Safe account on Neon EVM Mainnet"